lane.py

```diff
@@ -672,18 +672,27 @@ def modify_database_query(self, _db, qu):
         # Apply any superclass criteria
         qu = super(Facets, self).modify_database_query(_db, qu)
 
+        available_now = or_(
+            LicensePool.open_access == True,
+            LicensePool.self_hosted == True,
```

Comment:
What does it mean to be "self_hosted" and that it's "available now"?

Reply:
`self_hosted` books are books that are not open access, but were uploaded to a content server under the control of the library and are available without restriction to authenticated patrons. Right now that means they were uploaded to an LCP server and made available through an LCP collection.

This comes out of Viacheslav's work on LCP. As part of that work he changed a number of places where we assumed a book was "available" only because it's open access or because there are a certain number of licenses owned. This is a place that we missed originally.